### PR TITLE
Re-add configuration for resource requests and limits, remove kubernetes manifest

### PR DESCRIPTION
**Pull request by szEvEz**
- The resource configuration moved out, due to usage of the helmify command - I have also not found a way to configure it the way we need it
- removing manifests in deploy/ so we do not need to maintain two sets of config - we could think about automating it

From my point of view there is also no real benefit of storing the deploy manifests in the repo, if there is manual configuration required (adapting url and secret). If we could get around that, I'd love to generate a "deployment.yaml" and attach it to the release, like cert-manager does it for instance.

### CODE REVIEW

**Comment by rndmh3ro** [comment]
> removing manifests in deploy/ so we do not need to maintain two sets of config - we could think about automating it

Well, it was automated. :)

**Comment by szEvEz** [comment]
> > removing manifests in deploy/ so we do not need to maintain two sets of config - we could think about automating it
> 
> Well, it was automated. :)

But generating the helm chart from the deployment manifests using helmify is currently dropping configuration. How do we want to proceed here?

**Comment by rndmh3ro** [comment]
> > > removing manifests in deploy/ so we do not need to maintain two sets of config - we could think about automating it
> > 
> > 
> > Well, it was automated. :)
> 
> But generating the helm chart from the deployment manifests using helmify is currently dropping configuration. How do we want to proceed here?

I'd be fine with generating the deploy-files from the helm-charts or modifying helmify so it does not drop the configuration.

NB: not everybody uses (or likes) helm and I want to support that. Removing the static files would also be a breaking change.

**Comment by szEvEz** [comment]
> Hm, you completely removed the documentation on how to use it without helm. Can you re-add this please?

Sure, done!

File: README.md [hunk 1]
```diff
@@ -48,14 +48,23 @@ helm repo update
 helm install chart-name trivy-dojo-report-operator/trivy-dojo-report-operator --values your_values.yaml
 ```
 
-* Deploy the chart manually after cloning the git-repository:
+* or Deploy the chart manually after cloning the git-repository:
```
Comment (rndmh3ro) [inline README.md:51]:
```suggestion
* or deploy the chart manually after cloning the git-repository:
```

File: README.md [hunk 2]
```diff
@@ -48,14 +48,23 @@ helm repo update
...
 
 ```bash
 git clone https://github.com/telekom-mms/trivy-dojo-report-operator.git
 cd trivy-dojo-report-operator
 helm install chart-name charts/
 ```
 
+* or Deploy the trivy-dojo-report-operator using the Kubernetes manifests
```
Comment (rndmh3ro) [inline README.md:59]:
```suggestion
* or deploy the trivy-dojo-report-operator using the Kubernetes manifests
```

**Review by rndmh3ro — APPROVED** [review]

**Pull request merged**